kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/ErrorMessageTest.java

```diff
@@ -51,16 +44,18 @@ public void testCustomMessage() {
       .endItem().build()).once();
     server.expect().withPath("/api/v1/namespaces/test/events/event1").andReturn(403, Boolean.FALSE).once();
 
-    KubernetesClient client = server.getClient();
 
-    client.v1().events().inNamespace("test").delete();
+    try{
+      client.v1().events().inNamespace("test").delete();
+    } catch (Exception e){
+      Assertions.assertThat(e.getMessage().startsWith("Failure executing: DELETE"));
+      Assertions.assertThat(e.getMessage().contains("Message: MSG"));
+      Assertions.assertThat(not(e.getMessage().contains("Received status")));
+    }
```

Comment:
Should use assertThrows, or at the very least have a `fail();` statement to make sure that the Exception is thrown and assertions are passed.

Reply:
Done!